identity/app/views/support/fragment/ConsentChannel.scala

```diff
@@ -0,0 +1,33 @@
+package views.support.fragment
```

Comment:
We use 2 spaces indentation in our Scala code.

Reply:
huh not sure how that slipped by